website/src/craftml/ScriptGenerator.jsx

```diff
@@ -18,12 +21,22 @@ export function generateScript(elements) {
 
     let elementScript = '\t<element' + String(elementId) + '\n';
     elementScript += '\tmodule="' + currElement.module + '"\n';
-    // Add in transitions
-    elementScript += '\tt="size x ' + String(currElement.size.width / 10);
-    elementScript += ' y ' + String(currElement.size.height / 10) + '; ';
-    elementScript += 'position x ' + String(currElement.position.x / 10);
-    elementScript += ' y ' + String(currElement.position.y / 10) + '; ';
-    elementScript += 'rotate z ' + String(currElement.rotation.toFixed()) + '" />';
+
+    // Add in transitions based on orientation
+    if (orientation === SIDE_VIEW) {
```

Comment:
Instead of replicating all of the code in each of the blocks, can you just set whether we are using y or z as a variable then insert it into the code?